doc/nrf/gs_programming.rst

```diff
@@ -212,7 +212,11 @@ After completing the :ref:`manual <build_environment_cli>` or :ref:`automatic <g
 
       .. note::
 
-	     To build from a directory other than the application directory, run the west build command with an additional parameter *directory_name*,  specifying the application directory.
+         To build from a directory other than the application directory, run the west build command with an additional parameter *directory_name*,  specifying the application directory.
+
+         To get extra device tree overlays with new features available for a board version, run the west build command with an additional parameter *@board_revision*.
+         The *board_revision* is printed on the label of your DK, just below the PCA number.
+         For example, if you run the west build command with an additional parameter *@0.14.0* for nRF9160 build target, it adds the external flash on the nRF9160 DK that was available since board version 0.14.0.
 
```

Comment:
@gregersrygg @umapraseeda : could you please confirm whether this note is appropriate.
```suggestion
         You can run the west command with different parameters:
         
* *build_target* *directory_name* - To build from a directory other than the application directory, use *directory_name* to specify the application directory.

* *build_target@board_revision* - To get extra device tree overlays with new features available for a board version. The *board_revision* is printed on the label of your DK, just below the PCA number. For example, if you run the west build command with an additional suffix *@1.0.0* for the nRF9160 build target, it adds the external flash on the nRF9160 DK that was available since board version 0.14.0.
```

Reply:
@divipillai @umapraseeda Could you please remove `*build_target*` before `*directory_name*`? And perhaps say _optional_ instead of _different_?

The `*directory_name*` is part of the `west build -h` help text, so I'm not sure why we duplicate it here instead of referring to `west build -h` for extra build arguments. The down side of the help text is that it's limited space available in the command line help text, so we can't use as many words as in the docs. What do you think?
<img width="510" alt="source_dir help text" src="https://user-images.githubusercontent.com/149725/156730567-2af08efa-c745-4c7e-b2ff-2758b8b42684.png">

The `@board_revision` command line help text has been merged upstream, but it won't be in nRF Connect SDK before v2.0.0.

Reply:
We will remove the *build_target* before *directory_name* and say 'optional' instead of 'different'.
 
We would retain the parameter and its explanation and can add a mention about west build -h for other extra build arguments.